src/lib/actions/sendEvent/createSendEvent.js

```diff
@@ -10,12 +10,18 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-module.exports = ({ instanceManager, turbine }) => settings => {
+module.exports = ({
+  instanceManager,
+  turbine,
+  decisionsCallbackStorage
+}) => settings => {
   const { instanceName, ...otherSettings } = settings;
   const instance = instanceManager.getInstance(instanceName);
 
   if (instance) {
-    instance("sendEvent", otherSettings);
+    instance("event", otherSettings).then(decisions => {
```

Comment:
I think this should be reverted to `sendEvent`.

Reply:
yes, thank you, that is a mistake